appData/src/gb/src/Scene_b.c

```diff
@@ -587,7 +587,7 @@ void SceneUpdateActors_b()
     {
       if (ACTOR_ANIM_SPEED(ptr) == 4 || (ACTOR_ANIM_SPEED(ptr) == 3 && IS_FRAME_16) || (ACTOR_ANIM_SPEED(ptr) == 2 && IS_FRAME_32) || (ACTOR_ANIM_SPEED(ptr) == 1 && IS_FRAME_64) || (ACTOR_ANIM_SPEED(ptr) == 0 && IS_FRAME_128))
       {
-        if (ACTOR_MOVING(ptr) || ACTOR_ANIMATE(ptr))
+        if ((ACTOR_MOVING(ptr) & actors[i].sprite_type != SPRITE_STATIC) || ACTOR_ANIMATE(ptr))
```

Comment:
I think this is supposed to be a logical AND `&&` rather than a bitwise one `&`

Reply:
Good spotting, i'll change that up.
Just checked because i was curious, switching to logical `&&` seems a hair faster than bit wise & here, in bgb cpu.